view/ViewableData.php

```diff
@@ -541,6 +542,9 @@ public function CSSClasses($stopAtClass = 'ViewableData') {
 			$classes[] = $this->template;
 		}
 
+		// Strip out namespaces
+		$classes = preg_replace('#.*\\\\#', '', $classes);
```

Comment:
This function has nothing to do with namespaced templates. It takes a PHP class as an argument, and returns a list of CSS classes to use to hook CSS selectors on to. So I don't understand your comment, sorry.